sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample09_ObservableEventBatch.md

```diff
@@ -2,6 +2,17 @@
 
 This sample demonstrates how to write an `ObservableEventDataBatch` class that wraps an `EventDataBatch` in order to allow an application to read events that have been added to a batch. This is unlike the standard `EventDataBatch` class which restricts the application from accessing `EventData` instances once they have been accepted into the batch. `EventDataBatch` has this limitation to ensure that the state of the batch remains consistent and valid as events are added, and that it can successfully be published.
 
+## Table of contents
+
+- [Considerations](#considerations)
+- [Approach](#approach)
+    - [Benefits](#benefits)
+    - [Trade-offs](#trade-offs)
+    - [Using the Observable Data Batch](#using-the-observable-data-batch)
+    - [Note on Client Lifetime](#note-on-client-lifetime)
+        - [Accessing the EventData Instances](#accessing-the-eventdata-instances)
+        - [Comparing Identity](#comparing-identity)
+
```

Comment:
It looks like we messed up headings when we wrote this.    Could we please promote the items from L11-14 to the same header level as `Approach`?

Reply:
Sure thing. This has been fixed.